scheduler/__init__.py

```diff
@@ -1,2 +1 @@
 # coding=utf-8
-default_app_config = 'scheduler.apps.SchedulerConfig'
```

Comment:
Why removing it? It defined translated verbose name, using e. g. in admin backend.

Reply:
See commit message: 

> Remove default_app_config
> 
> RemovedInDjango41Warning:
> Django now detects this configuration automatically. You can remove default_app_config.
>